cmd/rest/client.go

```diff
@@ -117,5 +149,43 @@ func NewClient(url *url.URL, newCustomTransport func() *http.Transport, newAuthT
 		httpIdleConnsCloser: tr.CloseIdleConnections,
 		url:                 url,
 		newAuthToken:        newAuthToken,
+		connected:           online,
+
+		MaxErrResponseSize:  4096,
+		HealthCheckPath:     "",
+		HealthCheckInterval: 200 * time.Millisecond,
+		HealthCheckTimeout:  time.Second,
```

Comment:
do we want this dynamic or no? based on some sampling of previous attempts?

Reply:
No, not really. Even if a host has been down 10 minutes we still want to reconnect as fast as possible IMO. 5 checks/second should be ok to handle.